stl/inc/ranges

```diff
@@ -1042,6 +1042,10 @@ namespace ranges {
             return _STD addressof(*_Val);
         }
 
+        _NODISCARD static constexpr bool empty() noexcept {

```

Comment:
Stylistic nitpick: We prefer to follow the Standard's declaration order unless there's a good reason otherwise. (This makes it easier to compare the Standardese against our implementation.) The issue resolution declares `empty()` between `end()` and `size()` above.